### DISCUSSION TITLE
Fluxcd 2 with git-secret

### DISCUSSION THREAD

**Discussion by amitsikka** [fluxcd/flux2#4299] [Q&A] [answered]
Does fluxcd 2 support git-secret native same as fluxcd

**Comment by stefanprodan** [comment, marked as answer]
We only support [SOPS](https://fluxcd.io/flux/guides/mozilla-sops/) natively. Decryption happens now in-memory before the apply, in a diffrent controller (kustomize-controller) to the one (source-controller) that pulls data from Git, so it's not possible to use git-secret or any other solution like that with Flux v2.
